src/config.rs

```diff
@@ -133,6 +138,18 @@ impl Config {
             config.excerpt_separator = excerpt_separator.to_owned();
         };
 
+        #[cfg(all(feature="syntax-highlight", not(windows)))]
+        {
+            if let Some(theme) = yaml["syntax-highlight"]["theme"].as_str() {
+                if has_theme(theme) {
+                    config.theme = theme.to_owned();
+                } else {
+                    warn!("Theme named '{}' is not loaded. Using default 'base16-ocean.dark'.",
+                          theme);
+                }
+            };
+        }
+
```

Comment:
TravisCI will catch the `warn!` and fail?  If not, then I'd rather `has_theme` returns false and generates the non-fatal warning.

I'm opposed to sending mixed signals to users.  If a Windows user sets the `theme` field in `.cobalt.yml`, I don't want to give them an indication that cobalt used their syntax theme and then they see the code blocks on their site aren't highlighted.

Another option is to change the `warn!` to `info!` if TravisCI complains about the former and ignores the latter.  It reduces the visibility of the message, but it's a trade-off worth making if the user is informed with cobalt's actual behavior.

If neither option is suitable, I'll defer and update `has_theme` to return false.